browser/brave_wallet/brave_wallet_utils.cc

```diff
@@ -0,0 +1,72 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
```

Comment:
Nit: Might want to update these to 2020 now.

Reply:
updated all new instances of 2019 to 2020. 👍